megarules.py

```diff
@@ -259,51 +276,55 @@ def add_co_event_lar(event_rule, projectName, AOI_name, workflow, workflow_versi
     logger.debug(mode+"rule added")
     print(mode+"rule added")
 
+    add_user_rules.submit_iterator_job(projectName, rule_name, workflow_name, priority, event_rule, other_params)
+    # submit_jobs(rule_name, workflow, workflow_version, other_params, event_rule, dataset_tag)
     return rules_info
```

Comment:
`submit_iterator_job `does 2 things:
1. Add GRQ user rules for associated job
2. Submit an iterator job described by the rules in (1).
     i.e. for LARs: Facet on the interested SLCPs, and create LARs for each SLCP. (1:1)